schema/materials_category/schemas_categorization.yaml

```diff
@@ -0,0 +1,254 @@
+#
```

Comment:
If we want to merge this file, we need to put more explanation at the top here - what it represents, why we have it here (temporary), and we need to update the content to reflect the actual file structure